scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/job/ClientJobState.java

```diff
@@ -59,12 +58,12 @@ public ClientJobState(JobState jobState) {
 
         this.maxNumberOfExecution = jobState.getMaxNumberOfExecution();
 
-        this.genericInformation = new HashMap<>(jobState.getGenericInformation());
-
         this.clientJobSerializationHelper = new ClientJobSerializationHelper();
 
         this.setOnTaskError(jobState.getOnTaskErrorProperty().getValue());
 
+        this.genericInformation = jobState.getGenericInformation();
```

Comment:
I'm scared there is a reason for this defensive copy, are you sure it is not needed ?